sources/subs/InlinePermissionsForm.class.php

```diff
@@ -47,7 +47,7 @@ public static function save($permissions)
 			foreach ($_POST[$permission] as $id_group => $value)
 			{
 				if (in_array($value, array('on', 'deny')) && (empty($context['illegal_permissions']) || !in_array($permission, $context['illegal_permissions'])))
-					$insertRows[] = array((int) $id_group, $permission, $value == 'on' ? 1 : 0);
+					$insertRows[] = array('id_group' => (int) $id_group, 'permission' => $permission, 'add_deny' => $value == 'on' ? 1 : 0);
```

Comment:
The parameters now need reordered

		$insertRows[] = array($permission, (int) $id_group, $value == 'on' ? 1 : 0);

Array keys are ignored.